lntest/itest/lnd_trackpayments_test.go

```diff
@@ -0,0 +1,108 @@
+package itest
+
+import (
+	"context"
+	"encoding/hex"
+
+	"github.com/btcsuite/btcd/btcutil"
+	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/lightningnetwork/lnd/lnrpc/routerrpc"
+	"github.com/lightningnetwork/lnd/lntest"
+	"github.com/stretchr/testify/require"
+)
+
+// testTrackPayments tests whether a client that calls the TrackPayments api
+// receives payment updates.
+func testTrackPayments(net *lntest.NetworkHarness, t *harnessTest) {
+	// Setup two connected nodes.
+	alice := net.NewNode(t.t, "alice", nil)
```

Comment:
nit: could use `net.Alice` and `net.Bob` instead, which saves us from sending coins, making connections, etc.

Reply:
Done. I hope the cleanup is good, it's only closing the channel between `net.Alice` and `net.Bob`

Reply:
Yeah it actually just creates Alice and Bob every time a new case runs, and I'm fixing the itest.